src/applications/debt-letters/components/DebtDetails.jsx

```diff
@@ -85,26 +85,22 @@ class DebtDetails extends Component {
               </div>
             </div>
 
-            {nextStep ? (
-              <div className="debt-details-nextstep">{nextStep}</div>
-            ) : (
-              <AlertBox
-                className="vads-u-margin-y--4 debt-details-alert"
-                headline="Lorem ipsum dolor sit"
-                content={
-                  <>
-                    <p>
-                      <strong>Lorem ipsum dolor sit.</strong> Ducimus, maxime.
-                      Maxime neque alias unde totam beatae repellat aliquam
-                      ipsam, quas tempore nostrum et facere? Nulla nisi dolorum
-                      maiores molestiae fugiat.
-                    </p>
-                  </>
-                }
-                status="info"
-                level={2}
-              />
-            )}
+            {additionalInfo &&
+              (additionalInfo.nextStep ? (
+                <div className="debt-details-nextstep">
+                  {additionalInfo.nextStep}
```

Comment:
Would it save some work to include 

```html
 <strong>Next step: </strong>
```

here instead of repeated in the `additionalInfo` content?

Reply:
nah unfortunately cuz i have to keep the next step content pure since its being consumed in other places without the label